src/derivative_operators/concretization.jl

```diff
@@ -13,7 +13,7 @@ function Base.copyto!(L::AbstractMatrix{T}, A::DerivativeOperator{T}, N::Int) wh
 
     coeff   = A.coefficients
     get_coeff = if coeff isa AbstractVector
-        i -> coeff[i]
+        i = get_coefficient(coeff, i)

```

Comment:
Do you reference the variable `i` before it's defined here? Get an error `julia> Array(CenteredDifference{1}(2, 3, 0.01, 100, c_squared))
ERROR: UndefVarError: i not defined
Stacktrace:
 [1] copyto!(::Array{Float64,2}, ::DerivativeOperator{Float64,1,false,Float64,StaticArrays.SArray{Tuple{5},Float64,1,5},StaticArrays.SArray{Tuple{1},StaticArrays.SArray{Tuple{5},Float64,1,5},1,1},Array{Float64,1},Array{Float64,1}}, ::Int64) at /home/simen/Documents/school/masteroppgave/wave_simulation/dev/DiffEqOperators/src/derivative_operators/concretization.jl:16
 [2] Array(::DerivativeOperator{Float64,1,false,Float64,StaticArrays.SArray{Tuple{5},Float64,1,5},StaticArrays.SArray{Tuple{1},StaticArrays.SArray{Tuple{5},Float64,1,5},1,1},Array{Float64,1},Array{Float64,1}}, ::Int64) at /home/simen/Documents/school/masteroppgave/wave_simulation/dev/DiffEqOperators/src/derivative_operators/concretization.jl:45 (repeats 2 times)
 [3] top-level scope at REPL[2]:1` when using a locally merged version of this pull request.

Reply:
Changed te `=` to `->` in line 16 as the following lines, which works for me.